index.html

```diff
@@ -72,15 +72,21 @@ <h1 class="title">Spatialization demo</h1>
         <div id="controls">
           <button id="toggleCam"></button>
           <button id="toggleMic"></button>
+          <button
+            id="toggleScreenShare"
+            class="screen-disabled hidden"
+            disabled
+          ></button>
```

Comment:
Accessibility-wise, it would be better if these buttons had content in it that describes the action.

Reply:
Good point - would this be primarily for screen readers? (ie, would it work to set the text in the code but then have it be invisible in the UI itself?) Thanks for pointing this out.

Reply:
Yes, it's just for screen readers. We could use `aria-label` instead of content, so we don't have to hide it in a weird CSS way.

Reply:
Done!